examples/inputs/turbine_definitions/nrel_5MW.yaml

```diff
@@ -0,0 +1,155 @@
+turbine_type: 'nrel_5MW'
+generator_efficiency: 1.0
+hub_height: 90.0
```

Comment:
Could potentially move this to the main input file (see comment above).

Reply:
See above comment.